SI/Sem.03/Pract.03/Task02.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+int main() {
+	int n = 1;
+	std::cin >> n;
+	for (int i = 1; i < n; i++) {
+		for (int j = n - 1; j > 0; j--) {
```

Comment:
Think about how you can solve the task without nested loop

Reply:
#include <iostream>
int main() {
	int n = 1, addend = 1;
	std::cin >> n;
	addend = n;
	for (int i = 1; i < n; i++) {
		addend--;
		std::cout << "x = " << i << " y = " << addend << std::endl;
	}
}